internal/testing/e2e/docker.go

```diff
@@ -17,6 +19,10 @@ type dockerImageContext struct{}
 
 var dockerImageContextKey = dockerImageContext{}
 
+const (
+	envvarExtraDockerImages = envvarPrefix + "DOCKER_IMAGES"
```

Comment:
this lets me use a custom build of consul enterprise locally by building out-of-band and then specifying:

```
E2E_APIGW_CONSUL_IMAGE=consul-dev:ent
E2E_APIGW_DOCKER_IMAGES=consul-dev:ent
```
in the test run

Reply:
This functionality doesn't seem to be working for me in local testing, not quite sure for what reason yet - whether there's actually an issue here or if I'm perhaps skipping a step somewhere. @nathancoleman any luck validating this?